Documentation/project-docs/dogfooding.md

```diff
@@ -138,7 +138,7 @@ and run a self-contained application you need the following steps to consume you
 
 #### 1 - Get the Version number of the CoreFx package you built.
 
-Look for a package named `Microsoft.Private.CoreFx.NETCoreApp.<version>.nupkg` under `corefx\bin\packages\Debug` (or Release if you built a release version of corefx).
+Look for a package named `Microsoft.Private.CoreFx.NETCoreApp.<version>.nupkg` under `corefx\bin\packages\Debug` or `corefx\artifacts\packages\Debug`  (or Release if you built a release version of corefx).
```

Comment:
This should just mention artifacts.

This documentation is meant to match the rest of the source. We do not try to make it work for the past. It would become incomprehensible if we have tried that.

Reply:
ok